src/components/Book.js

```diff
@@ -9,20 +9,49 @@ const Book = ({ book }) => {
   };
 
   return (
-    <div className="book-details">
-      <p className="bk-title">
-        {' '}
-        {book.title}
-        {' '}
-      </p>
-      <p className="bk-author">
-        {' '}
-        {book.author}
-        {' '}
-      </p>
-      <button type="button" onClick={handleRemoveBook}>
-        Remove
-      </button>
+    <div className="book-wrapper">
+      <div className="book">
+        <div className="book-detail">
+          <p className="bk-genre">
+            {' '}
+            {book.category}
+            {' '}
+          </p>
+          <h2 className="bk-title">
+            {' '}
+            {book.title}
+            {' '}
+          </h2>
+          <p className="bk-author">
+            {' '}
+            {book.author}
+            {' '}
+          </p>
+        </div>
+        <div className="bookBtn">
+          <button type="button"> Comments </button>
+          <button type="button" onClick={handleRemoveBook}>
+            Remove
+          </button>
+          <button type="button"> Edit </button>
+        </div>
+      </div>
+
+      <div className="bk-status">
+        <div className="pie-graph"> </div>
+        <div className="completed-percentage">
+          <p className="percentage">64%</p>
+          <p className="completed">Completed</p>
+        </div>
+      </div>
+
+      <div className="process">
+        <div className="curr-chapter">
+          <p className="completed">Current Chapter</p>
+          <p className="chapter">Chapter  17</p>
+        </div>
+        <button type="button">UPDATE PROCESS</button>
+      </div>
```

Comment:
- [ ] Kindly note that you are required to replicate the styling exactly as it is on the design. Kindly use the correct fonts and sizes for the texts as we have in the design.

- [ ] Kindly take note of the background color. The original design has a somewhat lighter grey color but yours is darker. Kindly fix that as well.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![wiwow](https://user-images.githubusercontent.com/36542986/218398281-0ef7cd1f-4389-4bd9-930f-0f3824722f5c.jpg) | ![euwo](https://user-images.githubusercontent.com/36542986/218398286-7c8820f1-adf6-4d3a-a028-d64432e99868.jpg)